.github/CONTRIBUTING.md

```diff
@@ -7,11 +7,11 @@
 Search [the issue tracker][issue tracker] to ensure that there is no similar open issue.
 If that is the case, [open a new issue][new issue]. Use the following templates:
 
-* For feature requests, refer [here](../docs/template-featureRequest.md).
-* For bug reports, refer [here](../docs/template-bugReport.md).
-* For new contributor introduction, refer [here](../docs/template-contributorIntro.md).
-* For help requests, refer [here](../docs/template-helpRequest.md).
-* For other work like refactoring, updating documentation, or performance improvement, simply provide a clear **issue title** and **description** of the work to be done.
+* For bug reports, refer [here](template-bugReport.md).
+* For feature/enhancement requests, refer [here](template-featureRequest.md).
+* For new contributor introduction, refer [here](template-contributorIntro.md).
+* For help requests, refer [here](template-helpRequest.md).
+* For other works like refactoring or updating documentation, simply provide a clear issue title and description of the works to be done.
```

Comment:
`works` -> `work`
In this context, the singular "work" should still be used even though it might be referring to multiple subjects.